package.json

```diff
@@ -1,13 +1,23 @@
 {
-  "name": "@f1v/redliner",
-  "version": "0.0.1",
+  "name": "react-redliner",
+  "version": "0.0.64",
   "description": "A React utility to display redlines",
+  "keywords": [
+    "react",
+    "redliner",
+    "measure",
+    "style",
+    "attributes"
+  ],
   "main": "lib/index.js",
   "scripts": {
     "build:rollup": "tsc --project ./tsconfig.json && yarn rollup",
-    "build": "docz build",
+    "build": "rm -rf ./lib && tsc",
     "clean": "rm -rf .docz",
+    "format": "pretty-quick --staged",
+    "prepare": "yarn format && yarn build && yarn sass",
     "rollup": "rollup ./lib/RedLiner/RedLiner.js --file ./lib/index.js --format iife",
+    "sass": "sass ./src/components/RedLiner/RedLiner.module.scss ./lib/components/RedLiner/RedLiner.css && sass ./src/components/Line/Line.module.scss ./lib/components/Line/Line.css",
```

Comment:
Probably want a better system, however this is working to create the actual css files.